### DISCUSSION TITLE
关于nb-cli构建nbv2项目的问题

### DISCUSSION THREAD

**Discussion by WRTY1** [nonebot/discussions#366] [问答]
1.对于BIlibili Adapter的适配有问题，目前如果是自动构建而非手动构建的项目，会在`nb run --reload`这步跳出异常，然后卡死。
2.对于自动构建的项目，想自定义功能，是否只有通过自动构建插件模板后移动至项目文件这一条路？
3.对于插件，是否 具有方法直接调试，而非启动nbv2?
4.请问如何给机器人的插件添加代理？

**Comment by CMHopeSunshine** [comment]
1没用过不清楚，可以试试不加`--reload`
2没太看懂），直接在项目下新建插件进行功能编写就行
3可以先不编写与nb相关的部分，将核心功能部分进行调试，例如请求网络api等
4如果插件没有提供代理配置项的话，要开启全局代理，nb本身不负责这件事
